efopen/newrelic_executor.py

```diff
@@ -113,13 +113,20 @@ def update_cloudfront_policy(self):
     # Update Cloudfront alert policies
     logger.info("update cloudfront alert policy")
     cloudfront = self.clients['cloudfront']
-    distribution_list = cloudfront.list_distributions()['DistributionList']
+    pages = cloudfront.get_paginator('list_distributions').paginate()
+    queue = []
     map_function = lambda x: (x['Id'], ', '.join(x['Aliases']['Items']))
-    queue = map(map_function, distribution_list['Items'])
-
-    while distribution_list['IsTruncated']:
-      distribution_list = cloudfront.list_distributions(Marker=distribution_list['NextMarker'])['DistributionList']
-      queue.extend(map(map_function, distribution_list['Items']))
+    pages = cloudfront.get_paginator('list_distributions').paginate()
+    filtered_4xx_distributions = []
+    for page in pages:
+      for distribution in page['DistributionList']['Items']:
+        tag_response = cloudfront.list_tags_for_resource(Resource=distribution['ARN'])
+        # Remove from distribution list those that do not have tag nr_monitoring set to enabled
+        for tag in tag_response['Tags']['Items']:
+          if tag['Key'].lower() == "newrelic_4xx" and tag['Value'].lower() == "enabled":
```

Comment:
I probably worded this the wrong way during the last review.

Let's keep monitoring in the tag name.
```suggestion
          if tag['Key'].lower() == "newrelic_4xx_monitoring" and tag['Value'].lower() == "enabled":
```